packages/setup-remote-server/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "@wearerequired/setup-remote-server",
+  "version": "0.1.0-alpha.0",
+  "description": "Setup environment on remote server for deployment.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wearerequired/js.git"
+  },
+  "keywords": [
+    "cli",
+    "remote",
+    "server",
+    "environment"
+  ],
+  "author": {
+    "name": "required gmbh",
+    "email": "info@required.com",
+    "url": "https://required.com"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/wearerequired/js/issues"
+  },
+  "homepage": "https://github.com/wearerequired/js/tree/master/packages/setup-remote-server#readme",
+  "dependencies": {
+    "apache-md5": "^1.1.7",
+    "colorette": "^1.2.1",
+    "commander": "^6.0.0",
+    "conf": "^10.0.1",
+    "dotenv": "^10.0.0",
+    "inquirer": "^8.0.0",
+    "is-valid-hostname": "^1.0.2",
+    "node-ssh": "^12.0.0",
+    "ora": "^5.0.0",
+    "replace-in-file": "^6.1.0",
+    "untildify": "^4.0.0",
+    "yaml": "^1.10.2"
+  },
+  "engines": {
+    "node": ">=12.0"
+  },
+  "bin": {
+    "setup-remote-server": "./bin/setup-remote-server.js"
```

```suggestion
    "wearerequired-setup-remote-server": "./bin/setup-remote-server.js"
```

To better align with [@wearerequired/generate](https://github.com/wearerequired/js/blob/1b069332cbfe1621c1df915193b66f8755aaac35/packages/generate/package.json#L44). This makes the script basically available as `wearerequired-setup-remote-server` when installed globally.